test/validation/api/pktio/pktio.c

```diff
@@ -1480,6 +1481,43 @@ static void pktio_test_info(void)
 	}
 }
 
+static void pktio_test_link_info(void)
+{
+	odp_pktio_t pktio;
+	odp_pktio_link_info_t link_info;
+	int i;
+
+	for (i = 0; i < num_ifaces; i++) {
+		pktio = create_pktio(i, ODP_PKTIN_MODE_QUEUE,
+				     ODP_PKTOUT_MODE_DIRECT);
+		CU_ASSERT_FATAL(pktio != ODP_PKTIO_INVALID);
+
+		CU_ASSERT_FATAL(odp_pktio_link_info(pktio, &link_info) == 0);
```

Comment:
memset(&link_info, 0, sizeof(link_info))

would ensure that each successful call really fills in the info